### PR TITLE
Monthly report page for a selected project

### PR DESCRIPTION
**Pull request by YohannesTesfay**

### CODE REVIEW

File: app/frontend/components/reportForm/reportForm.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      <div className="flex flex-col">
+        <div className="flex justify-between">
+          <div>
+            <Combobox value={selectedProject} onChange={setSelectedProject}>
+              <div>
+                <div className="relative w-full cursor-default overflow-hidden rounded-lg bg-white text-left shadow-md focus:outline-none focus-visible:ring-2 focus-visible:ring-white focus-visible:ring-opacity-75 focus-visible:ring-offset-2 focus-visible:ring-offset-teal-300 sm:text-sm">
+                  <Combobox.Input<'input', ProjectFragment>
+                    className="w-full border-none py-2 pl-3 pr-10 text-sm leading-5 text-gray-900 focus:ring-0"
+                    displayValue={(project) => project.title}
+                    onChange={(event) => setProjectQuery(event.target.value)}
```
Comment (PascalHelbig) [inline app/frontend/components/reportForm/reportForm.tsx:47]: Does the combobox work without the `onChange` here?

File: app/frontend/components/topNavigation/topNavigation.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ export const TopNavigation = (): JSX.Element => {
       <TopNavigationLink href="/time">Time</TopNavigationLink>
       {teamSlug && <TopNavigationLink href={`/${teamSlug}/time`}>Time</TopNavigationLink>}
       {teamSlug && <TopNavigationLink href={`/${teamSlug}/projects`}>Projects</TopNavigationLink>}
-      <TopNavigationLink href="/reports">Reports</TopNavigationLink>
+      {teamSlug && <TopNavigationLink href={teamSlug ? `/${teamSlug}/reports` : '/reports'}>Reports</TopNavigationLink>}
```
Comment (PascalHelbig) [inline app/frontend/components/topNavigation/topNavigation.tsx:18]: We already check beforehand if there is a team slug. This if-condition is therefore unnecessary.
```suggestion
      {teamSlug && <TopNavigationLink href={`/${teamSlug}/reports`}>Reports</TopNavigationLink>}
```

File: app/frontend/components/reportForm/reportForm.tsx (hunk 1 above)
Comment (YohannesTesfay) [reply app/frontend/components/reportForm/reportForm.tsx:47]: It works, but I think the `OnChange` is required for `Combobox.Input`.

File: app/frontend/components/topNavigation/topNavigation.tsx (hunk 2 above)
Comment (YohannesTesfay) [reply app/frontend/components/topNavigation/topNavigation.tsx:18]: That, I really didn't notice 😊!

**Review by PascalHelbig — APPROVED** [review]
Looks good!

**Comment by YohannesTesfay** [comment]
Thank you!

**Pull request merged**